client/layout/sidebar/style.scss

```diff
@@ -3,6 +3,7 @@
 	// margins and paddings.
 	margin: 0;
 	padding: 0;
+	padding-top: 6px;
```

Comment:
I found that this small padding on top of the sidebar makes things a bit better when there's no upsell or notice on top of the menu.